librad/src/paths.rs

```diff
@@ -46,6 +47,7 @@ impl Paths {
         Self {
             keys_dir: config_dir.join("keys"),
             git_dir: data_dir.join("git"),
+            includes_dir: config_dir.join("includes"),
```

Comment:
More aptly might be `git-includes` as these are specific to git repositories.

Reply:
Mmm that's a good point. Which actually makes me wonder, should the includes path just be a child of the git dir?

Reply:
Was wondering the same, two points:

* I see the includes as config not as data
* having more writers to the monorepo path might not be desirable

Reply:
Fair points :+1: Happy with a separate path, and happy with `git-includes`